api/envoy/config/filter/network/http_connection_manager/v2/http_connection_manager.proto

```diff
@@ -453,6 +453,61 @@ message HttpConnectionManager {
   // with `prefix` match set to `/dir`. Defaults to `false`. Note that slash merging is not part of
   // `HTTP spec <https://tools.ietf.org/html/rfc3986>` and is provided for convenience.
   bool merge_slashes = 33;
+
+  // // Determines if adjacent slashes in the path are merged into one before any processing of
+  // // requests by HTTP filters or routing. This affects the upstream *:path* header as well. Without
+  // // setting this option, incoming requests with path `//dir///file` will not match against route
+  // // with `prefix` match set to `/dir`. Defaults to `false`. Note that slash merging is not part of
+  // // `HTTP spec <https://tools.ietf.org/html/rfc3986>` and is provided for convenience.
+  // bool merge_slashes = 33;
```

Comment:
What's going on here in this commented out block?

Reply:
My bad. Reverted